src/main/kotlin/com/ampnet/web3provider/exception/ErrorResponse.kt

```diff
@@ -0,0 +1,12 @@
+package com.ampnet.web3provider.exception
+
+data class ErrorResponse(
```

Comment:
ErrorResponse should be in the defined format with `code`, `message` and `data`. 
https://www.jsonrpc.org/specification#response_object

Reply:
The value of `data` is defined by the server. It is different in case of different providers. See for instance https://infura.io/docs/ethereum#operation/eth_getBalance or https://docs.alchemy.com/alchemy/documentation/error-reference.  I can map it with Any?

Reply:
Set `data` to `Any?` -  A Primitive or Structured value that contains additional information about the error. This may be omitted.

Whole error response should look like:
```json
{
  jsonrpc: String,
  error: {
    code: Int,
    message: String,
    data: Any?
  },
  id: Any?
}
```

Reply:
Yeah I just realised that result (in case of positive) can be a string or an object so I have to make some changes over there as well.